test/e2e/test-amp-bind-brightcove.js

```diff
@@ -19,10 +19,10 @@ describes.endtoend(
   {
     fixture: 'amp-bind/bind-brightcove.html',
   },
-  async (env) => {
+  (env) => {
     let controller;
 
-    beforeEach(async () => {
+    beforeEach(() => {
```

Comment:
I don't think it's necessary here, but I believe karma (mocha?) can support async before/after hooks without issue

Reply:
Yeah, unnecessary `async`s in `beforeEach()` blocks are likely harmless, but I've gone ahead and removed them wholesale in #35672.